OpenBB-finance/OpenBBTerminal
# Handle bad network connections

# Description

This PR handles SSLErrors and RequestExceptions happening inside the terminal. It replaces the bad error message with a prettier one. People can still replace these error messages with their own by handling the issues before they get to the decorator.


# How has this been tested?
Given that the PR just adds more detailed exception handling I did not see a need to test extensively. However, I did do the following:
- [x] Make sure affected commands still run in terminal
    - Turn off computer wifi
    - `python terminal.py stocks`
- [x] Ensure the SDK still works
    - Turn off computer wifi
    - `from openbb_terminal.sdk import openbb`


# Checklist:

- [x] Update [our Hugo documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).
- [x] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [x] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [x] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/scripts).


# Others
